scalableminds/webknossos
# don’t run the periodic cleanup jobs on shutdown (where they failed)

the cleanup jobs can’t run on shutdown anyway, because the db connection is already closing

### Steps to test:

- run locally with sbt
- change something in the scala code, so that F5 triggers hot reloading
- console output should feature `Executing Global END` but not `Exception during execution of cleanup job`

- to test that periodic cleanup execution still works, reduce period in Global.scala to something very short, test if console output shows `Completed cleanup job`


### Issues
- fixes #2732

------
